src/App.js

```diff
@@ -1,13 +1,11 @@
 import React, { Component } from "react";
 import fire from "./config";
 import Login from "./components/LoginRegister";
-import InputForm from "./components/InputForm";
 import "./App.css";
-import ReportHistory from "./components/ReportHistory";
-import history from "./history";
-import { Router } from "react-router-dom";
 import Routes from "./Routes";
 
+let DEV_URL="https://medical-requistion.herokuapp.com";
```

Comment:
Maybe go with this to prevent the url from mutation.
```suggestion
const DEV_URL="https://medical-requistion.herokuapp.com";
```